src/Avalonia.Input/KeyGesture.cs

```diff
@@ -61,6 +61,16 @@ public override int GetHashCode()
             return !Equals(left, right);
         }
 
+        static public implicit operator KeyGesture ((Key key, KeyModifiers modifiers) value)
```

Comment:
Adding a implicit operators is a [pretty big deal](https://stackoverflow.com/questions/1782503/implicit-operator-when-is-it-a-good-bad-idea) and should be discussed separately.